app/shared/video-player/common/ui/guesture/GestureLock.kt

```diff
@@ -92,23 +94,23 @@ fun GestureLock(
  */
 @Composable
 fun LockedScreenGestureHost(
-    controllerVisible: () -> Boolean,
-    setControllerVisible: (visible: Boolean) -> Unit,
+    controllerVisible: () -> VisibleState,
+    setControllerVisible: (visible: VisibleState) -> Unit,
```

Comment:
```suggestion
    setControllerVisibility: (visibility: VisibleState) -> Unit,
```

将属性从 boolean 修改为多元后, 也需要对应修改它的用法变量名